### ISSUE TITLE
ProperNouns with colon

### ISSUE THREAD

**Issue by thegoatherder** [spencermountain/compromise#998]
From what I can gather, Compromise assumes any `Title Cased` words are `#ProperNoun` unless they are the start of a sentence (or paragraph).

I think other punctuation like a colon `:` should prevent `#ProperNoun` tagging. Consider the text: `Management Plan: Review in 2 weeks` - Compromise tags `Plan` and `Review` as `#ProperNoun` even though there is a colon between them.

```javascript
const nlp = require('compromise')
console.log(nlp('Management Plan: Review in 2 weeks').debug())
```

Output
```ascii
'Management'  - Noun, Singular
  │ 'Plan'     - ProperNoun, Noun
  │ 'Review'   - Noun, Singular, ProperNoun
  │ 'in'       - Preposition
  │ '2'        - Value, Cardinal, NumericValue
  │ 'weeks'    - Noun, Plural
```

**Comment by thegoatherder** [comment]
@spencermountain any idea when this will be released pls?

**Comment by spencermountain** [comment]
release is currently blocked behind some breaking changes to efrt. Probly next week.

**Comment by thegoatherder** [comment]
@spencermountain  I did some testing on this today. I'm getting some mixed results.

```javascript
import nlp from './src/four.js'
nlp.verbose('tagger')

// ✅ this works
nlp('Management Plan: Review in 2 weeks').debug()

// ❌ This one still is tagging `Review` as a `#ProperNoun`
nlp('Management Plan: Review telephone consultation in three months').debug()
```

**Comment by spencermountain** [comment]
released as 14.9.0
